GETTING_STARTED.md

```diff
@@ -0,0 +1,30 @@
+The simplest way to start to use 'AppKid' is an example app like:
+
+https://github.com/smumriak/AppKidDemo
+
+...which you can [after installing dependances] simply git-clone to your local machine, and 'swift run' it.  
```

Comment:
You can use \`\` quotes to highlight code blocks like the `swift run` you have

Reply:
done